scripts/update_xcm_json_file.py

```diff
@@ -0,0 +1,231 @@
+    asset,
+    assetLocationPath,
+    destination: Optional[XcmTransfer] = None
+) -> Asset:
+
+    if (destination is None):
+        destination = build_xcm_transfer(base_parameters)
+
+    new_asset = Asset(
+        assetId=asset.assetId,
+        assetLocation=base_parameters.asset,
+        assetLocationPath=assetLocationPath,
+        xcmTransfers=[destination]
+    )
+
+    return new_asset
+
+
+def create_new_network(base_parameters: BaseParameters, destination: Optional[XcmTransfer] = None) -> XcmJson:
+    xcm_object = XcmJson(**xcm_json)
+
+    network_param = prompt(new_network_questions())
+
+    searched_chain = find_chain(chains_json, base_parameters.source_network)
+    asset = find_asset_in_chain(searched_chain, base_parameters.asset)
+
+    new_asset = create_new_asset(
+        base_parameters,
+        asset,
+        network_param.get('assetLocationPath'),
+        destination
+    )
+
+    network = Network(searched_chain.chainId, new_asset)
+
+    xcm_object.chains.append(network)
+
+    xcm_object.networkBaseWeight[network.chainId] = str(
+        network_param.get('networkBaseWeight'))
+
+    return xcm_object
+
+
+def push_new_destination(base_parameters: BaseParameters, xcm_transfer: XcmTransfer) -> XcmJson:
+
+    xcm_object = XcmJson(**xcm_json)
+
+    searched_chain = find_chain(chains_json, base_parameters.source_network)
+
+    for chain in xcm_object.chains:
+        if (chain.chainId == searched_chain.chainId):
+            for asset in chain.assets:
+                if (asset.assetLocation == base_parameters.asset):
+                    asset.xcmTransfers.append(xcm_transfer.__dict__)
+
+    return xcm_object
+
+
+def update_destinations(base_parameters: BaseParameters, destination: Optional[XcmTransfer] = None) -> XcmJson:
+
+    if (token_already_added(base_parameters)):
+        if (destination is None):
+            destination = build_xcm_transfer(base_parameters)
+        return push_new_destination(base_parameters, destination)
+    else:
+        xcm_object = XcmJson(**xcm_json)
+
+        asset_location = prompt(asset_location_question())
+
+        searched_chain = find_chain(
+            chains_json, base_parameters.source_network)
+
+        asset = find_asset_in_chain(searched_chain, base_parameters.asset)
+
+        new_asset = create_new_asset(
+            base_parameters=base_parameters,
+            asset=asset,
+            assetLocationPath=asset_location.get('assetLocationPath'),
+            destination=destination
+        )
+
+        for chain in xcm_object.chains:
+            if (chain.chainId == searched_chain.chainId):
+                chain.assets.append(new_asset)
+
+        return xcm_object
+
+
+def main():
+    '''
+    That script uses for update data in xcm/../transfers.json file.
+
+    To begin you must choose:
+    1. The asset which you want to add
+        - if that asset haven't added yet in the json file, you should add it manually in assetsLocation part.
```

Comment:
In this case we need to provide reserveFee manually?

Reply:
yes, it's still must be done manually. I think it would not be a problem to implement it in closer future.